lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -63,6 +77,20 @@ def scrub(transcript)
 
       private
 
+      def void_type(option = nil)
+        return 'payments/' unless option.present?
+
+        void_type_valid?(option) ? "#{option}/" : ''
+      end
+
+      def void_type_valid?(type)
+        %w(payments captures refunds credits).include?(type)
+      end
+
+      def eligible_for_zero_verify?(payment_method, options = {})
```

Comment:
⚠️ warning:

To avoid code/duplication if you are going to use `eligible_for_zero_verify` method from CybersourceGateway, copy that method on CyberSourceCommon module, we are going to remove the methods included there from CybersourceGateway in a future PR

Reply:
Done!